optimizely/optimizely_config.py

```diff
@@ -136,9 +117,121 @@ def __init__(self, project_config):
         self.environment_key = project_config.environment_key
         self.attributes = project_config.attributes
         self.events = project_config.events
+        self.rollouts = project_config.rollouts
 
         self._create_lookup_maps()
 
+        '''
+            Merging typed_audiences with audiences from project_config.
+            The typed_audiences has higher presidence.
+        '''
+
+        typed_audiences = project_config.typed_audiences[:]
+        optly_typed_audiences = []
+        for typed_audience in typed_audiences:
+            optly_audience = OptimizelyAudience(
+                typed_audience.get('id'),
```

Comment:
I think you are right, should be able to use a map instead of doing line 141 - creating a list and then checking length of returned values. Good thinking :)